test/extensions/filters/http/header_to_metadata/config_test.cc

```diff
@@ -99,6 +110,37 @@ TEST(HeaderToMetadataFilterConfigTest, PerRouteConfig) {
   EXPECT_FALSE(config->doResponse());
 }
 
+TEST(HeaderToMetadataFilterConfigTest, ValueAndRegex) {
```

Comment:
Please add a short comment describe what the test case is checking. (e.g. "// Tests that configuration does not allow value and regex_value_rewrite in the same rule."). Same for the other tests.